language/transaction-builder/src/generated.rs

```diff
@@ -5,94 +5,697 @@
 //
 // To re-generate this code, run: `(cd language/stdlib && cargo run --release)`
 
+//! Conversion library between a structured representation of a Move script call (`ScriptCall`) and the
+//! standard LCS-compatible representation used in Libra transactions (`Script`).
+//!
+//! This code was generated by compiling known Script interfaces ("ABIs") with the tool `transaction-builder-generator`.
+
+#![allow(unused_imports)]
 use libra_types::{
     account_address::AccountAddress,
     transaction::{Script, TransactionArgument},
 };
 use move_core_types::language_storage::TypeTag;
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeMap as Map;
+
+/// Structured representation of a call into a known Move script.
+/// ```ignore
+/// impl ScriptCall {
+///     pub fn encode(self) -> Script { .. }
+///     pub fn decode(&Script) -> Option<ScriptCall> { .. }
+/// }
+/// ```
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, PartialOrd)]
+pub enum ScriptCall {
+    /// Add a `Currency` balance to `account`, which will enable `account` to send and receive
```

Comment:
So awesome that the comment formatting is preserved!

Reply:
The auto-wrap that I added earlier was not a great idea.